.circleci/config.yml

```diff
@@ -353,7 +297,6 @@ workflows:
       - deployment-checks: *release-branches
       - integration-tests-cocoapods: *release-tags-and-branches
       - integration-tests-swift-package-manager: *release-tags-and-branches
-      - integration-tests-carthage: *release-tags-and-branches
```

Comment:
we actually should probably run all of these tests on the branch since it makes structural changes, just to make sure that everything still works afterwards on all package managers

Reply:
✅